src/components/DropDownMenu/index.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+interface DropDownMenuProps {
+  menuContent: string[];
+}
+const DropDownMenu: React.FC<DropDownMenuProps> = ({ menuContent }) => {
+  return (
+    <ul className="pl-[20px] pt-[20px]">
+      {menuContent.map((item, index) => (
+        <li key={index} className="paragraph--p2 pb-[20px] hover:text-black">
```

Comment:
add `text-[#828282] hover:text-[#333]`

Reply:
by the way, padding 20px can be written `p-5`

Reply:
Hi Shawn,

The issues you mentioned in this PR has been fixed. Thanks.